README.md

```diff
@@ -1,77 +1,166 @@
-# time_managment_tool
+# Time Management Tool 
 
-This project is for managing the time and prioritizing the task
 
-### Setup Instructions
+## Project Origin
 
-#### 1. Fork & clone the Repository.
+This project was inspired by a personal journey shared in the tweet: [View Original Tweet](https://x.com/NikhilEth/status/1846957571761983971)
 
-Fork the [repository](https://github.com/AthenaFoss/time_managment_tool) & clone the repository using -
+The idea emerged from a desire to create a comprehensive time management tool that goes beyond traditional tracking methods, focusing on productivity and personal growth.
+
+![Time Management App Preview](https://pbs.twimg.com/media/GaG2H34WcAAgcee?format=jpg&name=large)
+
+## Overview
+
+A comprehensive web application designed to help you manage time, prioritize tasks, and boost productivity. Built with modern web technologies and inspired by advanced time management techniques.
+
+
+## Key Features
+
+### Productivity Arsenal 💪🎯
+- 🔥 Smart Task Prioritization (Urgent, Important, Time-taken)
+- 🍅 Customizable Pomodoro Timer
+- 📋 Intelligent To-do Lists
+- 📊 Performance Analytics Dashboard
+
+### PWA Superpowers 🌐📱
+- 🚫 Offline Mode Support
+- 💻 Desktop & Mobile Installation
+- 🔔 Smart Push Notifications
+- 🔄 Background Synchronization
+- 🏠 Home Screen Shortcut
+
+## Tech Stack
+
+- Frontend: Next.js
+- Backend: Node.js
+- Database: PostgreSQL
+- ORM: Prisma
+- Authentication: Google OAuth
+- Deployment: TBD
+
+## Prerequisites
+
+- Node.js (v16 or later)
+- npm
+- Docker (optional, for local database)
+- PostgreSQL
+
+## Environment Setup
+
+### 1. Clone the Repository
 
 ```bash
 git clone https://github.com/{your_github_username}/time_managment_tool.git
+cd time_managment_tool
 ```
 
-#### 2. Install Dependencies
-
-Open the `time_management_tool` directory in your preferred code editor or IDE.
-
-Open a terminal for the directory, then run the following command in the terminal to install all the dependencies.
+### 2. Install Dependencies
 
 ```bash
 npm install
 ```
 
-#### 3. Configure Environment Variables
+### 3. Configure Environment Variables
 
-In the root of the directory, create a `.env` file by copying the `.env.example` file.
+Create a `.env` file by copying the example:
 
 ```bash
 cp .env.example .env
 ```
 
-In the `.env` file, set up the application environment and authentication routes.
+Update the `.env` file with your configuration:
 
 ```
 NODE_ENV=development
+DATABASE_URL=postgresql://your_username:your_password@localhost:5432/time_management
 ```
 
```

Comment:
Looks good so far! Just mention that we have `"ui:add" in place of "npx shadcn@latest add"` as shadcn command to add the UI components, so that one can know that we have command optimizations.

For reference take a look on `package.json` and see this `"ui:add": "npx shadcn@latest add"`

Thanks!